dashboard/test/ui/features/teacher_tools/teacher_dashboard/age_gated_students_modal.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Age Gated Students Modal and Banner
+  @eyes
```

Comment:
Oh cool, I didn't realize you could have @eyes apply to just a single scenario in the `.feature`.